docs/developers/contributing/documentation/index.md

```diff
@@ -115,6 +115,16 @@ want to contribute. The paths are listed in parentheses below.
 - [**Getting started**](getting-started) (in [`napari/docs/tutorials/fundamentals`](https://github.com/napari/docs/tree/main/docs/tutorials/fundamentals)):
   these documents are a mix of tutorials and how-tos covering the fundamentals of installing and working with napari for beginners
 
+```{admonition} Plugin documentation
+:class: tip
+
+Some of the source files for the [Plugin documentation](plugins-index) are
+autogenerated from sources in the [napari/npe2](https://github.com/napari/npe2)
+repository. These files' names start with an underscore `_` and will be
+generated when the documentation is built and the [`prep_docs.py`](https://github.com/napari/docs/blob/main/docs/_scripts/prep_docs.py)
```

Comment:
Could we link to our Makefile, I didn't immediate get that `prep_docs.py` is used in our make file and runs when we build locally. Maybe add that these files are not in the `napari/docs` repo but get added when we build docs locally.

I know it may seem obvious but useful for newbies :pray: